src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.ts

```diff
@@ -279,6 +289,27 @@ export class ListComponent implements OnInit {
             this.clearSelectedItems();
             this.loadAll();
         });
+
+        this.eventSubscriber = this.eventManager.subscribe('germplasmSelectorSelected', (event) => {
+            const searchComposite = new SearchComposite<GermplasmSearchRequest, number>();
+            searchComposite.itemIds = event.content.split(',');
+            this.germplasmListService.addGermplasmEntriesToList(this.listId, searchComposite)
+                .pipe(finalize(() => {
+                    this.isLoading = false;
+                })).subscribe(
+                (res: void) => {
+                    this.eventManager.broadcast({ name: 'addToGermplasmList', content: this.listId });
+                    this.alertService.success('germplasm-list.list-data.add-entries.success');
+                },
+                (res: HttpErrorResponse) => this.onError(res)
+            );
+        });
+
+        this.eventSubscriber = this.eventManager.subscribe('addToGermplasmList', (event) => {
+            if (this.listId === event.content) {
+                this.load();
```

Comment:
Why does it need to have a separate addToGermplasmList event here? Can you directly call "this.load" inside germplasmSelectorSelected event?